tests/e2e-tests/specs/backend/single-product.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import {
+	insertBlock,
+	getAllBlocks,
+	switchUserToAdmin,
+} from '@wordpress/e2e-test-utils';
+
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+
+const block = {
+	name: 'Single Product',
+	slug: 'woocommerce/single-product',
+	class: '.wc-block-single-product',
+};
+
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
+		await switchUserToAdmin();
+		await visitBlockPage( `${ block.name } Block` );
+	} );
+
+	it( 'can be inserted more than once', async () => {
+		await insertBlock( block.name );
+		expect( await getAllBlocks() ).toHaveLength( 2 );
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Delete' );
+		expect( await getAllBlocks() ).toHaveLength( 1 );
```

Comment:
To not create conflict with other tests, we delete the newly created second block, I'd argue this test is not needed because all blocks can be inserted more than once by default.

Reply:
Ya the test is probably not needed.